Sources/VGSCollectSDK/Core/Collector/VGSCollect+network.swift

```diff
@@ -167,4 +167,25 @@ extension VGSCollect {
           block(response)
         }
     }
+  
+  public func tokenizeData(completion block: @escaping (VGSResponse) -> Void) {
```

Comment:
`VGSResponse` or `VGSRequestResult`

Reply:
we don't have VGSRequestResult object in SDK.